content/riak/kv/2.0.0/developing/data-types.md

```diff
@@ -30,7 +30,7 @@ data types: flags, registers, [counters](#counters),
 Of those five types, counters, sets, and maps can be used as
 bucket-level data types, i.e. types that you can interact with directly.
 Flags and registers, however, must be embedded in maps (more on that
-[below](/riak/kv/2.0.0/developing/data-types/#maps).
+[below](/riak/kv/2.0.0/developing/data-types/maps).
```

Comment:
In this and all 2.0.z docs: 

1. The note needs to say Riak KV rather than Riak.